pkg/crc/machine/machine.go

```diff
@@ -63,17 +64,18 @@ func getCrcBundleInfo(bundlePath string) (*bundle.CrcBundleInfo, error) {
 	return bundle.Extract(bundlePath)
 }
 
-func loadCrcBundleInfo(startConfig *StartConfig, bundleName string) (*bundle.CrcBundleInfo, error) {
-	crcBundleMetadata, err := bundle.GetCachedBundleInfo(bundleName)
-	if err != nil {
-		return nil, err
+func getBundleMetadataFromDriver(driver drivers.Driver) (string, *bundle.CrcBundleInfo, error) {
+	bundleName, err := driver.GetBundleName()
+	if bundleName == "" || err != nil {
```

Comment:
We should add a comment here about removing `bundleName== ""` condition after 3-4 release since after that it would be not needed.

Reply:
@cfergeau we already updated our machine drivers, when I said crc releases that means the user side when they get those updated machine drivers :) Like with this beta release user should have updated machine driver.

Reply:
For the latest machine-driver-libvirt (0.12.5), go.mod is https://github.com/code-ready/machine-driver-libvirt/blob/880ceaa894a0bc65d2db413af58fb827ff599c88/go.mod with machine commit `34d5b6049a38`. This commit is one commit _before_ the one adding the error when bundle name is empty (see https://github.com/code-ready/machine/commits/master ). So I don't think the current builds of the machine drivers have this code just yet.

Reply:
Ah :( we should've done it before this release. I will create PR for libvirt and hyperkit side to update it with latest machine code.